sql/moz-fx-data-shared-prod/firefox_desktop_derived/newtab_visits_daily_v2/schema.yaml

```diff
@@ -0,0 +1,169 @@
+fields:
```

Comment:
@jaredsnyder is the is the position required in table `newtab_visits_revenue_v1` in the model? If so, It'd be useful to have it in this `newtab_visits_daily` as it is the source for revenue one.